book/forms.rst

```diff
@@ -1122,6 +1122,23 @@ the choice is ultimately up to you.
             ));
         }
 
+.. caution::
+
+    When the name of your form class matches any of the built-in field types,
```

Comment:
This does not only apply to the built-in types, but is also true for any other form type that is coming from third-party bundles/libraries.